presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -115,7 +115,11 @@ public static void assertBlockEquals(Type type, Block actual, Block expected)
 
     public static Block createAllNullsBlock(Type type, int positionCount)
     {
-        return new RunLengthEncodedBlock(type.createBlockBuilder(null, 1).appendNull().build(), positionCount);
+        BlockBuilder blockBuilder = type.createBlockBuilder(null, 1);
```

Comment:
unrelated change? -- since the original RLE block is also a valid representation for all nulls block?

Reply:
It just feels a little contrintuitive. I would expect the `createAllNullsBlock` to create a native block for a given type when all positions are nulls. If the intent is to create a RLE block with a null value - there should be a separate method that does that.

Reply:
Let me extract it into a separate commit